src/widgets/lv_btnmatrix.c

```diff
@@ -232,6 +235,10 @@ void lv_btnmatrix_clear_btn_ctrl(const lv_obj_t * obj, uint16_t btn_id, lv_btnma
 
     btnm->ctrl_bits[btn_id] &= (~ctrl);
     invalidate_button_area(obj, btn_id);
+
+    if (ctrl & LV_BTNMATRIX_CTRL_POPOVER) {
+        lv_obj_refresh_ext_draw_size(obj);
```

Comment:
CI doesn't like this, because `obj` is `const lv_obj_t`, but the API expects `lv_obj_t`.

@kisvegabor What is the purpose of `const` on this API, since it modifies the control bits?

Reply:
> What is the purpose of const on this API, since it modifies the control bits?

It seems like a mistake. `lv_btnmatrix_set_btn_ctrl` has non `const` parameter and `lv_btnmatrix_clear_btn_ctrl` should have too.

Reply:
Good catch. I removed the `const` as part of this PR now.